airbyte-db/db-lib/src/main/java/io/airbyte/db/mongodb/MongoUtils.java

```diff
@@ -73,6 +74,7 @@ public class MongoUtils {
   public static final String MONGODB_CLUSTER_URL = "mongodb+srv://%s%s/%s?retryWrites=true&w=majority&tls=true";
   public static final String MONGODB_REPLICA_URL = "mongodb://%s%s/%s?authSource=admin&directConnection=false&ssl=true";
   public static final String USER = "user";
+  public static int MAX_LEVEL_FROM_CONFIG=2;
```

Comment:
Rename this variable to what it means and not from what it comes from. What the default 2 means?

Reply:
by default it goes upto 2 levels of depth